qiskit_ibm_runtime/_cli.py

```diff
@@ -0,0 +1,234 @@
+        """Print lines in a box using Unicode box-drawing characters"""
+        width = max(len(line) for line in lines)
+        box_lines = [
+            "╭─" + "─" * width + "─╮",
+            *(f"│ {Format.bold(line.ljust(width))} │" for line in lines),
+            "╰─" + "─" * width + "─╯",
+        ]
+        print("\n".join(box_lines))
+
+    @classmethod
+    def get_channel(cls) -> Channel:
+        """Ask user which channel to use"""
+        print(Format.bold("Select a channel"))
+        return select_from_list(["ibm_quantum", "ibm_cloud"])
+
+    @classmethod
+    def get_token(cls, channel: Channel) -> str:
+        """Ask user for their token"""
+        token_url = {
+            "ibm_quantum": "https://quantum.ibm.com",
+            "ibm_cloud": "https://cloud.ibm.com/iam/apikeys",
+        }[channel]
+        print(
+            Format.bold("\nPaste your API token")
+            + f"\nYou can get this from {Format.cyan(token_url)}."
+            + "\nFor security, you might not see any feedback when typing."
+        )
+        while True:
+            token = getpass("Token: ").strip()
+            if token != "":
+                return token
+
+    @classmethod
+    def get_instance(cls, service: QiskitRuntimeService) -> str:
+        """
+        Ask user which instance to use, or select automatically if only one
+        is available.
+        """
+        instances = service.instances()
+        if len(instances) == 1:
+            instance = instances[0]
+            print(f"Using instance {Format.greenbold(instance)}")
+            return instance
+        print(Format.bold("\nSelect a default instance"))
+        return select_from_list(instances)
+
+    @classmethod
+    def save_to_disk(cls, account: dict) -> None:
+        """
+        Save account details to disk, confirming if they'd like to overwrite if
+        one exists already. Display a warning that token is stored in plain
+        text.
+        """
+        try:
+            AccountManager.save(**account)
+        except AccountAlreadyExistsError:
+            response = user_input(
+                message="\nDefault account already exists, would you like to overwrite it? (y/N):",
+                is_valid=lambda response: response.strip().lower() in ["y", "yes", "n", "no", ""],
+            )
+            if response.lower() in ["y", "yes"]:
+                AccountManager.save(**account, overwrite=True)
+            else:
+                print("Account not saved.")
+                return
+
+        print(f"Account saved to {Format.greenbold(_DEFAULT_ACCOUNT_CONFIG_JSON_FILE)}")
+        cls.print_box(
+            [
+                "⚠️ Warning: your token is saved to disk in plain text.",
+                "If on a shared computer, make sure to revoke your token",
+                "by regenerating it in your account settings when finished.",
+            ]
+        )
+
+
+def user_input(message: str, is_valid: Callable[[str], bool]) -> str:
```

Comment:
Maybe worth grouping these user input helper functions as static methods in a `UserInput` class. I like the grouping you're doing because it makes this single-file program better organized.